<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutRoot">
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <property name="spacing">
     <number>20</number>
    </property>

    <!-- Top spacer -->
    <item>
     <spacer name="verticalSpacerTop">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>

    <!-- Welcome label centered -->
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to bank!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>

    <!-- Form area centered with Grid -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutCenter">
      <item>
       <spacer name="horizontalSpacerLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>

      <item>
       <layout class="QGridLayout" name="gridLayoutForm">
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>16</number>
        </property>

        <!-- Row 0: card id -->
        <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>card id</string>
          </property>
         </widget>
        </item>

        <item row="0" column="1">
         <widget class="QLineEdit" name="textCardId">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>45</height>
           </size>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>

        <!-- Row 1: password -->
        <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>password</string>
          </property>
         </widget>
        </item>

        <item row="1" column="1">
         <widget class="QLineEdit" name="textPincode">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>45</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>

       </layout>
      </item>

      <item>
       <spacer name="horizontalSpacerRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>

    <!-- Login button centered -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButton">
      <item>
       <spacer name="horizontalSpacerBtnLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>

      <item>
       <widget class="QPushButton" name="btnLogin">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sign in</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>

      <item>
       <spacer name="horizontalSpacerBtnRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>

    <!-- Info label (centered, wraps long messages) -->
    <item>
     <widget class="QLabel" name="labelInfo">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>

    <!-- Bottom spacer -->
    <item>
     <spacer name="verticalSpacerBottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>

 </widget>
 <resources/>
 <connections/>
</ui>
